tests/dto/thirdparty.test.ts

```diff
@@ -592,13 +592,18 @@ describe('thirdparty', () => {
     const thirdPartyRequestsVerificationsPostRequest: Schemas.ThirdpartyRequestsVerificationsPostRequest = {
       verificationRequestId: correlationId,
       challenge: 'some challenge base64 encoded',
-      value: {
-        authenticationInfo: {
-          authentication: 'U2F',
-          authenticationValue: 'some signed challenge base64 encoded'
-        }
-      },
-      consentId: correlationId
+      consentId: '8d34f91d-d078-4077-8263-2c0498dhbjr',
```

Comment:
aya...meaning we need to add fetching and creating dfsp backend calls for this id.